File: codingTest/goorm/Java/1103_정사각형.java
//https://level.goorm.io/exam/49086/%EC%A0%95%EC%82%AC%EA%B0%81%ED%98%95%EC%9D%98-%EA%B0%9C%EC%88%98/quiz/1

import java.io.*;
class Main {
	public static void main(String[] args) throws Exception {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		String input = br.readLine();
		int square = Integer.valueOf(input);
		
		long result = 0L;
		for (long i = 1; i < square+1; i++) {
			//i*i가 int보다 크니까 i도 long으로 
			result += i*i;
		}
		System.out.println(result);
	}
}


//i*i말고, i의 n제곱근을 구하는 Math.pow(i, n)으로 풀어본 것

import java.io.*;
class Main {
	public static void main(String[] args) throws Exception {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		String input = br.readLine();
		int square = Integer.valueOf(input);
		
		long result = 0L;
		for (int i = 1; i < square+1; i++) {
			result += (long)Math.pow(i, 2);
		}
		System.out.println(result);
	}
}

File: codingTest/goorm/py/1103_정사각형.py
#https://level.goorm.io/exam/49086/%EC%A0%95%EC%82%AC%EA%B0%81%ED%98%95%EC%9D%98-%EA%B0%9C%EC%88%98/quiz/1

#파이썬3에서는 long이 없어지고 int의 범위가 더 넓어졌다고 한다.
#64비트보다 큰 데이터도 처리할 수 있지만, 그렇다고 무한대는 아니다.


# -*- coding: utf-8 -*-
# UTF-8 encoding when using korean
user_input = int(input())

def sqCount(n):
	result = 0;
	for i in range(1, n+1):
		result += i*i
	return result

print(sqCount(user_input))
